web-app/CKS.Web/wwwroot/404.html

```diff
@@ -0,0 +1,6 @@
+<html>
+	<head/>
+	<body>
+		<h1>Page not found</h1>
```

Comment:
Can we add some text in here that basically says that this file will be replaced by the contents of the gatsby build and that it is used for integration tests, just so someone doesn't get confused and they know what it's for?

Reply:
done